python/ray/utils.py

```diff
@@ -390,6 +390,35 @@ def setup_logger(logging_level, logging_format):
     logger.propagate = False
 
 
+def open_log(path, **kwargs):
+    kwargs.setdefault("buffering", 1)
+    kwargs.setdefault("mode", "a")
+    return open(path, **kwargs)
```

Comment:
I'd suggest `open(..., encoding="utf-8")` because on Windows this doesn't default to UTF-8 (which I presume the log files are assumed to be).

Reply:
Will do, has this not come up before though?

Reply:
It will once people start running Ray on Windows and non-ASCII characters get emitted into the log :-)